### PR TITLE
Add reply message to channel

### PR DESCRIPTION
**Pull request by hkhanouchi**
https://user-images.githubusercontent.com/27851097/214836893-2b152547-f4db-413a-9249-0bd880f44604.mov

### CODE REVIEW

File: src/components/message-input/styles.scss [hunk 1]
```diff
@@ -110,6 +110,46 @@
...
+  margin-bottom: 8px;
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2px 10px 0 0;
+  }
+  &__icon {
+    margin: 5px 5px 0 5px;
+    color: var(--accent-color);
```
Comment (mrval5) [inline src/components/message-input/styles.scss:125]: also theme, and the border below

File: src/components/message-input/reply-card.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { IconButton, Icons } from '@zer0-os/zos-component-library';
```
Comment (mrval5) [inline src/components/message-input/reply-card.tsx:1]: maybe better to make a new folder for `replay-card`

File: src/components/message-input/reply-card.test.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { shallow } from 'enzyme';
+import Dropzone from 'react-dropzone';
+
+import ReplyCard, { Properties } from './reply-card';
+
+describe('ReplyCardReplyCard', () => {
```
Comment (mrval5) [inline src/components/message-input/reply-card.test.tsx:8]:
```suggestion
describe('ReplyCard', () => {
```

**Review by mrval5 — CHANGES_REQUESTED** [review]

**Review by mrval5 — APPROVED** [review]

File: src/platform-apps/channels/messages-menu/index.tsx [hunk 4]
```diff
@@ -145,6 +157,7 @@ export class MessageMenu extends React.Component<Properties, State> {
           {menuItems}
         </PortalMenu>
         {this.showDeleteModal && this.renderDeleteModal()}
+        {this.showDeleteModal && this.renderDeleteModal()}
```
Comment (dalefukami) [inline src/platform-apps/channels/messages-menu/index.tsx:160]: This is suspicious to me. Rendering the delete modal twice?

**Pull request merged**